config/templates/pages/example.xml

```diff
@@ -196,6 +196,13 @@
             </meta>
         </property>
 
+        <property name="datetime" type="datetime">
```

Comment:
Please also add an output for that in the `example.html.twig` file

Reply:
added output for all missing properties